sktime/dists_kernels/compose_tab_to_panel.py

```diff
@@ -0,0 +1,77 @@
+# -*- coding: utf-8 -*-
+"""
+Composers that create panel pairwise transformers from table pairwise transformers
+"""
+
+__author__ = ["fkiraly"]
+
+import numpy as np
+
+from sktime.dists_kernels._base import BasePairwiseTransformerPanel
+
+
+class AggrDist(BasePairwiseTransformerPanel):
+    """
+    pairwise distance based on aggregation function applied to sample-wise distance
```

Comment:
Minor (potentially major?) comment:
I don't know what this sentence means. Could be me (I am not used to thinking about panel data) or could be sign docstring needs to be clearer

Reply:
I updated the docstring